### PR TITLE
README: "How it works" and "How to update"

### PR DESCRIPTION
**Pull request by tg666**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,13 +71,15 @@ Now you can open a preview of the website and as you can see the cookie widget i
 
 ## Configuration
 
+> :exclamation: We recommend locking the package version in the field `Package version` (see below) after you successfully configure the template. Avoid using the `latest` version in production containers!
```
Comment (jelen07) [inline README.md:74]:
```suggestion
> :exclamation: Locking the package version is highly recommended (see the field `Package version` below) after you successfully configure the template. Avoid using the `latest` version in production containers!
```

File: README.md [hunk 2]
```diff
@@ -256,6 +289,14 @@ The only currently available event is `init`. A callback is invoked when the wra
 </script>
 ```
 
+## How to update already published containers
+
+If you want to update to the newer version please firstly look into [releases](https://github.com/68publishers/cookie-consent/releases) to see what has changed until the release that you using.
```
Comment (jelen07) [inline README.md:294]:
```suggestion
If you want to update to the newer version please check out the [releases](https://github.com/68publishers/cookie-consent/releases) to see what has changed until the release that you using.
```

File: README.md [hunk 3]
```diff
@@ -14,14 +14,45 @@ An extended integration of [orestbida/cookieconsent](https://github.com/orestbid
...
 * [Integration into the GTM](#integration-into-the-gtm)
 * [Configuration](#configuration)
 * [Settings modal trigger](#settings-modal-trigger)
 * [Triggering tags based on the consent](#triggering-tags-based-on-the-consent)
 * [Accessing the wrapper in the JavaScript](#accessing-the-wrapper-in-the-javascript)
+* [How to update already published container](#how-to-update-already-published-containers)
 * [Development](#development)
 * [License](#license)
 
+## How it works
```
Comment (jelen07) [inline README.md:27]:
```suggestion
## How the GTM integration works
```

plus update the TOC

Comment (jelen07) [reply README.md:27]: Let's move this section before the `"How to update already published container"` section. I would like to include some "How it works" sections later, where will be not just the GTM part.

**Review by jelen07 — CHANGES_REQUESTED** [review]
Nice work!

File: README.md [hunk 4]
```diff
@@ -14,14 +14,45 @@ An extended integration of [orestbida/cookieconsent](https://github.com/orestbid
...
+    - :fast_forward: The [original plugin](https://github.com/orestbida/cookieconsent) is initialized
+    - :fire: Callbacks for an [event](#init-event) `CookieConsentWrappe.on('init')` are fired
+    - :eye: The consent modal is shown if a user has not yet agreed to the use of cookies
+
+### User updates his preferences through the setting modal
+
+- :arrows_counterclockwise: The consent update is sent into [native Google Consent API](https://developers.google.com/tag-platform/devguides/consent#gtag.js)
+- :fire: Custom triggers for newly granted storage types are fired
+
+<br>
```
Comment (jelen07) [inline README.md:54]: Is is necessary?

Comment (tg666) [reply README.md]: i removed it

File: README.md (hunk 3 above)
Comment (tg666) [reply README.md]: done

**Comment by tg666** [comment]
Okay, I incorporated the changes and rebase everything into one commit. I'm merging it...

**Pull request merged**